cpp/src/bindings.cpp

```diff
@@ -640,4 +640,43 @@ PYBIND11_MODULE(_pypowsybl, m) {
     m.def("remove_aliases", &pypowsybl::removeAliases, "remove specified aliases on a network", py::arg("network"), py::arg("dataframe"));
 
     m.def("close", &pypowsybl::closePypowsybl, "Closes pypowsybl module.");
+    /************************DYNAMIC SIMULATION WITH DYNAWALTZ*******************************/
+    //TODO descriptions of methods
```

Comment:
Actually docstrings here are not important, they are not exposed to the user.

About the separation comment above : I would propose to move everything to a dedicated function instead. We should probably have done it before to better organize the bindings.